appserver/tests/tck/cdi/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
```

Comment:
Just a question, but - what is the reason to use XML and not properties?

Reply:
Frankly? I have no idea, this one came in from the standalone runner.